packages/google_maps_flutter/lib/src/marker.dart

```diff
@@ -65,16 +79,9 @@ class MarkerController {
     String? newInfoWindowContent,
   }) {
     assert(_marker != null, 'Cannot `update` Marker after calling `remove`.');
+    _marker!.options = options;
     if (_infoWindow != null && newInfoWindowContent != null) {
       _infoWindow.content = newInfoWindowContent;
-      _infoWindow.pixelOffset = util.GSize(
-        (marker.infoWindow.anchor.dx - 0.5) * _markerWidth,
-        marker.infoWindow.anchor.dy * _markerHeight,
-      );
-    }
-    _marker!.options = options;
-    if (!marker.visible) {
-      hideInfoWindow();
     }
```

Comment:
Is there a reason for deleting it?

Reply:
Original source code does not have that part either.